l2gv2/induced_subgraph.py

```diff
@@ -1,23 +1,40 @@
+""" Induced subgraph utility functions. """
 import torch
 import torch_geometric as tg
 
+
 def induced_subgraph(data: tg.data.Data, nodes, extend_hops=0):
+    """ Induce a subgraph from the given nodes. 
+    
+    Args:
+        data: The original graph data.
+
+        nodes: The nodes to induce the subgraph from.
+        
+        extend_hops: The number of hops to extend the subgraph.
+
+    Returns:
+        The subgraph data object. 
+    """
+
     nodes = torch.as_tensor(nodes, dtype=torch.long)
     if extend_hops > 0:
-        nodes, edge_index, node_map, edge_mask = tg.utils.k_hop_subgraph(nodes, num_hops=extend_hops,
-                                                                         edge_index=data.edge_index,
-                                                                         relabel_nodes=True)
+        nodes, edge_index, _, edge_mask = tg.utils.k_hop_subgraph(
+            nodes, num_hops=extend_hops, edge_index=data.edge_index, relabel_nodes=True
+        )
         edge_attr = data.edge_attr[edge_mask, :] if data.edge_attr is not None else None
     else:
-        edge_index, edge_attr = tg.utils.subgraph(nodes, data.edge_index, data.edge_attr, relabel_nodes=True)
+        edge_index, edge_attr = tg.utils.subgraph(
```

Comment:
non-positive extend_hops behaviour can be documented

Reply:
@abhidg Shall we put TODO comments in the code to prompt the developers to add comments here and other similar code?